react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -396,10 +396,13 @@ const Map: React.FC<MapProps> = ({
 
     useEffect(() => {
         let tempViewStates: Record<string, ViewStateType> = {};
-        if (cameraPosition && Object.keys(cameraPosition).length !== 0) {
+        const isCameraPositionDefined =
```

Comment:
Not sure why do we need to introduce this const variable here ? I think the if statement before should be doing the same check ?

Reply:
I placed the variable in the top of the file so that it is calculated only once.